### ISSUE TITLE
Graphs inaccurate when large numbers involved

### ISSUE THREAD

**Issue by aeijdenberg** [grafana/grafana#11058]
We've noticed some strangeness occurring when there is reasonable size jump in the size of our metrics.

See the attached screenshot:
 
![image](https://user-images.githubusercontent.com/5984070/36881261-4f09cec8-1e21-11e8-9d63-20b0614a65dd.png)

While the graph to the right of the step appears to be correct, on the left, you can see from the legend displayed while we mouse over that the total should be well above 700 million, yet it shows as under 500 million.

If we zoom in on just that data to the left, we can see the scale shows the numbers as much higher:

![image](https://user-images.githubusercontent.com/5984070/36881400-02d88ab6-1e22-11e8-8713-198a512b5884.png)

(this is the same time period that displays as less than 500 million above.)

If I were to guess as to the issue, could be there be something that looks for the max value in a range, and uses that to calculate a "pixels per values" ratio, which is then multiplied against values before plotting? Perhaps when the numbers get this big, the underlying representation is truncated, resulting in too low a value being displayed?

We are using Grafana 4.6.3, and this particular metric is a stack graph, with around 70 values.

Appreciate any thoughts, and thanks for the great software.

**Comment by torkelo** [comment]
Please use the query inspector in the metrics tab to inspect the query sent and the response.

This is likely an issue with your query aggregation function , what data source you use ? What interval or aggregation does you query use?

**Comment by aeijdenberg** [comment]
TLDR; Set `Null value` to `null as zero` and everything is right again.

Behavior of a stack graph otherwise is weird, if interested, see smaller test-case below:

![image](https://user-images.githubusercontent.com/5984070/36893554-a0eff330-1e5c-11e8-8a07-cd3af65e8014.png)

And the data, note the `xenon2018` section has only one datapoint (which is an insignificant amount): 

```json
{
  "xhrStatus": "complete",
  "request": {
    "method": "GET",
    "url": "xxx"
  },
  "response": {
    "status": "success",
    "data": {
      "resultType": "matrix",
      "result": [
        {
          "metric": {
            "__name__": "processed_entries",
            "log": "ct.googleapis.com/icarus/"
          },
          "values": [
            [
              1519949323,
              "207913265"
            ],
            [
              1519949353,
              "207913265"
            ]
          ]
        },
        {
          "metric": {
            "__name__": "processed_entries",
            "log": "ct.googleapis.com/logs/xenon2018/"
          },
          "values": [
            [
              1519949353,
              "1001"
            ]
          ]
        },
        {
          "metric": {
            "__name__": "processed_entries",
            "log": "ct.googleapis.com/rocketeer/"
          },
          "values": [
            [
              1519949323,
              "220100443"
            ],
            [
              1519949353,
              "220100443"
            ]
          ]
        }
      ]
    }
  }
}
```

When `null as zero` is set, then the graph above changes to be flat (as it should).
